### PR TITLE
fix: update github.com/sabhiram/go-gitignore

### PR DESCRIPTION
**Pull request by denysvitali**
This change updates the `github.com/sabhiram/go-gitignore` dependency that was, weirdly enough, giving compatibility errors for not respecting the v1 contract.

### CODE REVIEW

File: actor/sharedaction/log_cache_client.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ import (
 	"time"
 
 	logcache "code.cloudfoundry.org/go-log-cache"
-	"code.cloudfoundry.org/go-loggregator/rpc/loggregator_v2"
+	"code.cloudfoundry.org/go-loggregator/v8/rpc/loggregator_v2"
```
Comment (a-b) [inline actor/sharedaction/log_cache_client.go:8]: Can you please elaborate on this proposed change?

**Comment by a-b** [comment]
Hi @denysvitali, can you please fix the EasyCLI check?

**Comment by denysvitali** [comment]
@a-b : sorry, since this contribution was made as part of my work-time, I would have to sign the CLA as a company contributor.

This will definitely take a lot of time due to the bureaucracy involved. I'll get back to you ASAP.

**Comment by a-b** [comment]
Hi @denysvitali do you have any updates?

File: actor/sharedaction/logging.go [hunk 2]
```diff
@@ -141,21 +141,23 @@ func GetStreamingLogs(appGUID string, client LogCacheClient) (<-chan LogMessage,
 		const offset = 1 * time.Second
 		walkStartTime := ts.Add(-offset)
 
+		v := func(envelopes []*loggregator_v2.Envelope) bool {
```
Comment (a-b) [inline actor/sharedaction/logging.go:144]: Also, can you please explain - what you are trying to achieve by moving this outside of the `Walk`?

**Comment by ccjaimes** [comment]
Given that this PR had no follow-up, I will proceed to close it. Feel free to reopen and we can revisit this discussion.

**Pull request closed without merging**